src/extra/utils/functions.py

```diff
@@ -25,7 +25,7 @@ def gaussian(x, y0, A, μ, σ, norm=True):
     return y0 + (A / norm_factor) * np.exp(-(x - μ)**2 / (2 * σ**2))
 
 
-def fit_gaussian(ydata, xdata=None, p0=None, norm=False, **kwargs):
+def fit_gaussian(ydata, xdata=None, p0=None, norm=False, Asign=0, **kwargs):
```

Comment:
Nitpicking, could we rename this to `A_sign`? I slightly prefer having the underscore for consistency with other possible parameters like `y0_min` etc.

Reply:
Sure, that makes sense.